tests/apollo/test_skvbc_preexecution.py

```diff
@@ -99,12 +107,12 @@ async def test_sequential_pre_process_requests(self, bft_network):
         """
         bft_network.start_all_replicas()
         skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+        await trio.sleep(SKVBC_INIT_GRACE_TIME)
 
         for i in range(NUM_OF_SEQ_WRITES):
             client = bft_network.random_client()
             await self.send_single_write_with_pre_execution(skvbc, client)
 
-    @unittest.skip("unstable")
```

Comment:
Please do not touch the disabled tests - they are out of scope for your PR.

Reply:
Sure. Undoing the chage.